### ISSUE TITLE
Editor losing focus when preview is invoked via keyboard shortcut

### ISSUE THREAD

**Issue by nameanyone** [Jermolene/TiddlyWiki5#2449]
See discussion at https://groups.google.com/d/msg/tiddlywiki/GlsruQyPOag/STvxOrR6BQAJ.
Does "don’t see a good solution" mean we'll have to live with it? Thought I'd document it here anyways.

**Comment by pmario** [comment]
I think this is kind of a dupe to: #2448 ... right? So at least the other 1 can be closed.

**Comment by rubaboo** [comment]
No, these are two very separate issues. This one is about editor losing focus when turning the preview on and off, the other is about the preview functionality not being readily available when using simple editor. (I'll add a comment there as well.)

**Comment by ldorigo** [comment]
Hey there, 

Just came to open an issue about this - its a small thing, but it's annoying and it would be nice to fix it. 

I understand it requires some work (as there is currently no mechanism to allow it), but implementing it could also be useful to for example make keyboard shortcuts that allow to go directly to the "edit" area of the tiddler (currently, after writing the tiddler title, the only way to start writing the tiddler body is to use the mouse - tabbing doesn't work (as it focuses the tag selector which makes the tags dropdown appear which then locks focus).

**Comment by pmario** [comment]
Reviewed .. OP It's still a problem .. still a bit annoying, but probably no easy fix. At least I don't know it. 

>... tabbing doesn't work (as it focuses the tag selector which makes the tags dropdown appear which then locks focus).

This problem is fixed in the meantime

**Comment by saqimtiaz** [comment]
This seems to be resolved in v5.3.6. Could you verify @pmario ?

**Comment by pmario** [comment]
@saqimtiaz -- I think this issue is resolved. @rubaboo, @Jermolene ... This issue can be closed
